packages/next/build/webpack/config/blocks/css/loaders/utils/base62.ts

```diff
@@ -0,0 +1,78 @@
+const dictionary = [
+  '0',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9',
+  'A',
```

Comment:
It's likely you want to switch the order here, starting with lowercase, then uppercase. There should be a benefit for compression size since it's more likely to be redundant with other content in the response.

Reply:
Thanks for the advice! What about numeric characters? Should I let them as is or should they be placed before or after uppercase?